src/decorators/withResources/createCallConfig.js

```diff
@@ -31,6 +32,13 @@ export default function createCallConfig(resource, defaultConfig) {
     config.params = Object.assign({}, config.params, filters)
   }
 
+  /**
+   * Setup Paging
+   */
+  if (Object.keys(paging).length > 0) {
+    config.params = Object.assign({}, config.params, paging)
```

Comment:
Sometimes `...` is used, something `Object.assign({}, x ,y)`, I think ideally we just choose one method to make shallow clones?
```suggestion
    config.params = { ...config.params, ...paging };
```

Reply:
Yeah, wanted to be consistent with the other assigns in this method.